{
    "content": [
        {
            "hidden": false,
            "name": "Force-the-use-of-the-system-cfitsio.patch",
            "stat": {
                "perms": "rw-r--r--",
                "size": 718,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "Mark-failures-that-are-known-to-fail-with-the-original-cf.patch",
            "stat": {
                "perms": "rw-r--r--",
                "size": 1143,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        },
        {
            "hidden": false,
            "name": "series",
            "stat": {
                "perms": "rw-r--r--",
                "size": 106,
                "symlink_dest": null,
                "type": "-"
            },
            "type": "file"
        }
    ],
    "directory": "patches",
    "package": "python-fitsio",
    "path": "python-fitsio/1.1.6+dfsg-1/debian/patches",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/python-fitsio/1.1.6+dfsg-1/",
        "metric": {
            "size": 864
        },
        "pts_link": "https://tracker.debian.org/pkg/python-fitsio",
        "sloc": [
            [
                "python",
                5425
            ],
            [
                "ansic",
                3474
            ],
            [
                "makefile",
                8
            ]
        ],
        "suites": [
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/debian-astro-team/python-fitsio",
        "vcs_type": "git"
    },
    "type": "directory",
    "version": "1.1.6+dfsg-1"
}